setup.py

```diff
@@ -31,10 +31,7 @@
 import glob
 import os
 import logging
-try:
-    from distutils.core import setup
-except ImportError:
-    from setuptools import setup
+from setuptools import setup, find_packages
```

Comment:
I'm not happy with introducing a hard requirement on `setuptools`, since it's not part of the Python standard library, and often includes backwards-incompatible changes in new versions...